Corrections for HCAL iterative phi symmetry calibration AlCaReco .py files

nilina opened the pull request:
3 python files were added for AlCaReco stream:
   Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIterativePhiSym_Output_cff.py
   Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIterativePhiSym_cff.py
   Calibration/HcalAlCaRecoProducers/python/alcaiterphisym_cfi.py

and 1python file for testing:
     Calibration/HcalAlCaRecoProducers/test/AlCaIterativePhiSym_cfg.py

No any corrections and changes for producers!

rovere committed:
Python macro to produce some interesting plots

rovere committed:
Add function to automatically compare old and new values

rovere committed:
Increase statistics

rovere committed:
Add command line options to dumpFullXML.py script.

rovere committed:
Extend functionalities to better trace differences generated by this procedure

rovere committed:
Remove unnecessary printouts

rovere committed:
Add dumping features that are TWiki friendly

rovere committed:
Fix plots names to be more meaningful

rovere committed:
Adde Eta stickers to the plots

rovere committed:
Add plots that show the distribution of values for energy loss and radiation length, as computed by the new procedure

cmsbuild commented:
A new Pull Request was created by @nilina  for CMSSW_7_5_X.

Corrections for HCAL iterative phi symmetry calibration AlCaReco .py files

It involves the following packages:

Calibration/HcalAlCaRecoProducers


@cmsbuild, @mmusich, @diguida, @cerminar, @nclopezo can you please review it and eventually sign? Thanks.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@nclopezo you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

cmsbuild commented:
+1
Tested at: e43ea4765fb4c722c07956037f316c145bfdd9f7
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/2812/summary.html

smorovic committed:
triggerTypeFlags bitmask (was reserved4) in TCDS header interface

smorovic committed:
lock --> unlock (not critical as process exits and releases the lock shortly after)

mmusich commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
-1 
Tested at: e43ea4765fb4c722c07956037f316c145bfdd9f7
 The relvals timed out after 1 hour. 
 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/3717/summary.html

diguida commented:
please test
The Frontier timeouts are now over

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: e43ea4765fb4c722c07956037f316c145bfdd9f7
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/4860/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/4860/summary.html

smorovic committed:
*added SSE 4.2 accelerated CRC32-C
 Original implementation by Mark Adler patched to compile as C++ and
 introduced ifdefs to compile SSE 4.2 asm version only for X86_64,
 otherwise using fallback to software crc32c implementation
*changes to support checking crc32c checksum in both raw input sources (V5 of the FRD format),
 included also support for V4 header
*ability to generate new FRD v5 format and checksum with fakeBU script

mmusich commented on Calibration/HcalAlCaRecoProducers/test/AlCaIterativePhiSym_cfg.py:
```diff
@@ -0,0 +1,39 @@
+
+process.load("Configuration.Geometry.GeometryIdeal_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```
please use the `condDBv2` version

mmusich commented on Calibration/HcalAlCaRecoProducers/test/AlCaIterativePhiSym_cfg.py:
```diff
@@ -0,0 +1,39 @@
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['startup']
```
please use the `condDBv2` version
The correct key for `autoCond_condDBv2` dictionary is `run2_mc`.

mmusich commented:
@nilina 
my understanding is that you want to add a new `ALCARECOHcalCalIterativePhiSym`, ALCARECO stream.
If you want this to be run in production you need to update:
`Configuration/EventContent/python/AlCaRecoOutput_cff.py`
`Configuration/EventContent/python/EventContent_cff.py`
and I guess `Configuration/StandardSequences/python/AlCaRecoStream_Specials_cff.py`

makortel committed:
Update track selector module labels to new iteration names

makortel committed:
Update validation plotting script for the new directory names

makortel committed:
Add README

makortel committed:
Use common track-TP association in track and vertex validations

makortel committed:
Migrate newly added configurations for VInputTag associators

Commit:
switch of PF patch, reshuffling Reconstruction_AftMix_cff.py

jhgoh committed:
efficiency Me name have been changed

Commit:
reducing the way to pass input to FamosProducer to the ways that are actually in use

Commit:
second take of: reducing the way to pass input to FamosProducer to the ways that are actually in use

Commit:
3rd take of: reducing the way to pass input to FamosProducer to the ways that are actually in use

venturia committed:
New scripts to monitor GlobalTags and noise ratios using conddb V2

aledegano committed:
BigObjs apply conditions for slc7 too.
The special compiler flags are needed for slc7_amd64_gcc49* too.
The change is trivial and doesn't change anything for current production architecture.

Commit:
remove never-used interfaces from FBaseSimEvent and FSimEvent

Commit:
remove template documentation

pietverwilligen committed:
reintroduce rotation except for RE+1/1

trocino committed:
New Medium muon ID selectors and offline DQM plots

venturia committed:
Create README.md

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

Commit:
Merged refs/pull/8875/head from repository cms-sw

Commit:
remove obsolete dependence of PF classes on FastSim vertex generator cfg

Commit:
put back the function FSimEvent::fill(SimTracks,SimVertices)

Commit:
remove FastSim VertexGenerator cfg files: they are no longer used

fcavallo committed:
pruning and minor changes

fcavallo committed:
restored DTLocalTrigerSynchTask

fcavallo committed:
restored DTResolutionTest

fcavallo committed:
restored DTResolutionTest

fcavallo committed:
restored DTResolutionTest

davidlange6 committed:
Merge pull request #8874 from degano/75X_bigobjs_slc7

BigObjs apply conditions for slc7 too.

davidlange6 committed:
Merge pull request #8859 from alja/roo6fw75

75 fireworks: port changes from 7_4_X, related to ROOT6 dictionaries

cmsbuild committed:
Merge pull request #8839 from dildick/GEM-static-analyzer-cleanup

Fix issues in GEM packages reported by static analyzer

Commit:
FWCore/Integration: remove unused variables (Clang errors)

Signed-off-by: David Abdurachmanov <David.Abdurachmanov@cern.ch>

ggovi committed:
Various fixes for ConddbEssource input parameters handling

apfeiffer1 committed:
fix printout-conversion of time-types (remove nsec part from Oracle/Frontier)

alja committed:
Merge commit 73782e8d6fb3bcf0061c5138a9c509f42b77a946 from CMSSW_6_2_X_SLHC branch.

alja committed:
Merge commit 73782e8d6fb3bcf0061c5138a9c509f42b77a946 and ed205f69a839dfb9883ef9b2f6371dcfa145afdc from CMSSW_6_2_X_SLHC branch.

alja committed:
Merge commit 73782e8d6fb3bcf0061c5138a9c509f42b77a946 and  ed205f69a839dfb9883ef9b2f6371dcfa145afdc from CMSSW_6_2_X_SLHC branch.

davidlange6 committed:
Merge pull request #8887 from apfeiffer1/conddb-fixTimeTypeFormat

fix conversion of time-types in printout

davidlange6 committed:
Merge pull request #8817 from wddgit/consumes103

Consumes migration, ExoticaDQM & HWWAnalyzer

makortel committed:
Update iteration names also in the plotting scripts

Commit:
new parameters for HcalResponse_cfi.py

geoff-smith committed:
hlt integrations for 75X

Dr15Jones committed:
Add nice pdf graph option to edmStreamStallGrapher

Optionally use matplotlib to create a nice pdf file of the stream stalls

geoff-smith committed:
uncommented CMSHLT-333 paths in subtables

Commit:
added Bertini Cascade for FastSimulation of nuclear interactions in tracker at low energy

alja committed:
Merge commit 73782e8d6fb3bcf0061c5138a9c509f42b77a946 and  235b639f6a18e94ebe0bc490a036834e902d39c2 from CMSSW_6_2_X_SLHC branch.

alja committed:
Add missing includes.

alja committed:
Merge commit ac4090c9aee63dd161136be41f7edf552d74d688 and 317fa70e27601a53839cf57d43539c9f49cd79bc from Sven in CMSSW_6_2_X_SLHC branch to 75.

Dr15Jones committed:
General improvements to edmStreamStallGrapher

-reduced size of generated PDF and added key
-improved help documentation
-formatted stalled module summary information

Dr15Jones committed:
Add accessors to SubProcess

We needed a way to access the Process held by the SubProcess in order for CRAB to fully probe a configuration.

cmsbuild committed:
Merge pull request #8903 from Dr15Jones/niceGraphicForStallGraph

Improvements to edmStreamStallGrapher

cmsbuild committed:
Merge pull request #8899 from alja/gemtra

75x Fireworks: port GEM related changes from CMSSW_7_6_X_SLHC

cmsbuild committed:
Merge pull request #8875 from lveldere/cleanup-famosproducer

FastSim clean FamosProducer

cmsbuild committed:
Merge pull request #8826 from VinInn/TrackRecovery

implement correct track breaking if propagation fails

smorovic committed:
use "Failsafe" as reserved destination name when sdestination can't be
calculated instead of empty string.

bendavid committed:
limit maximum energy (pt) for ecal pf cluster corrections to avoid bad extrapolation beyond region of correction derivation

jshlee committed:
Merged CMSSW_7_5_X from repository cms-sw

Commit:
correct name of test script

jshlee committed:
merged with updates

jshlee committed:
Merged CMSSW_7_5_X from repository cms-sw

jshlee committed:
adding me0

jshlee committed:
fix me0 in FWRPZViewGeometry.cc

jshlee committed:
Merged fireworks-gem-75x from repository jshlee

deguio committed:
Merged refs/pull/8786/head from repository cms-sw

deguio committed:
fixing compilation errors for PR #8786 and resubmitting

cmsbuild committed:
Merge pull request #8909 from lveldere/fastsim-rename-testEmptyHits

correct name of test script FastSimulation/Validation/test/testEmptyHits.sh

jshlee committed:
added me0 to FWRecoGeometryESProducer.cc

jshlee committed:
Merge branch 'fireworks-gem-75x' of https://github.com/jshlee/cmssw into fireworks-gem-75x
make

Commit:
Fix multiple booking in fillSummary.

perrotta committed:
Further HLT updates on top of CMSSW_7_4_1

capalmer85 committed:
first attempt with ALCARecos for vertex and pixel counting

puigh committed:
Update ele+HT paths to WPLoose

Commit:
FWCore/Services: do not optimize 'edmtest::FpeTester::analyze'

Resolves unit test (`fpe_test_2.sh`) failure in CLANG IB.

    Completed cmsRun with DIVIDEBYZERO exception enabled
    cmsRun status:  0
    Test FAILED, status neither 136 nor 11
    status = 256

`do_division` call is optimized out from `edmtest::FpeTester::analyze`.
It's caused by division by zero, which is undefined behavior thus
compiler is free to do what it wants, e.g., optimize it out.

Disable any optimizations on the function.

Signed-off-by: David Abdurachmanov <David.Abdurachmanov@cern.ch>

civanch committed:
cleanup gdm interface and R_hadron custom physics

deguio committed:
fix DQM test 4 (FS)

kencall committed:
This adds DQMOffline/Hcal and fixes the "severity typo"

Commit:
Updated implementation of the HCAL negative energy filter

Martin-Grunewald committed:
HLT in FastSim: fixes and use of standard cff dumps

cmsbuild committed:
Merge pull request #8921 from deguio/fixTest4a

fix DQM test 4 (FS)

cmsbuild committed:
Merge pull request #8920 from civanch/cleanup_gdml_and_customRhadrons

Cleanup gdml interface and R-hadron custom physics

cmsbuild committed:
Merge pull request #8919 from puigh/my-tthbb-triggerdqm-75x

Update ele+HT paths to WPLoose (75X)

cmsbuild committed:
Merge pull request #8911 from deguio/fixDT

Fix dt

cmsbuild committed:
Merge pull request #8898 from civanch/AddedBertiniCascadeForFastSim

Added Bertini Cascade for FastSimulation of nuclear interactions in tracker

cmsbuild committed:
Merge pull request #8892 from davidlt/fix-clang-FpeTester

FWCore/Services: do not optimize 'edmtest::FpeTester::analyze'

cmsbuild committed:
Merge pull request #8880 from davidlt/clang-clean-FWCore-Integration

FWCore/Integration: remove unused variables (Clang errors)

cmsbuild committed:
Merge pull request #8878 from venturia/toward_monitoring_withV2-75x

Update of the sistrip DB tags monitoring scripts and cfgs to be compliant with conddb V2

davidlange6 committed:
Merge pull request #8905 from Dr15Jones/addAccessorsToSubProcess

Add accessors to SubProcess

cmsbuild committed:
Merge pull request #8602 from jingyucms/jingyu-l1t-stage1-dqm-75x

Histogram for L1T Stage1

ggovi committed:
Reverting default configurations from condDB v1 to v2

cmsbuild committed:
Merge pull request #8881 from jhgoh/RPCRollEfficiencyBugfix

RPC efficiency distribution bugfix

cmsbuild committed:
Merge pull request #8895 from silviodonato/DQMIDTight

Path renaming in HLT DQM (JIRA-333) - 75X

cmsbuild committed:
Merge pull request #8858 from ebouvier/ElePahtsFromCMSHLT330ForTopValidation75X

Single electron paths from CMSHLT-330 for Top validation code in 75X

cmsbuild committed:
Merge pull request #8841 from cms-l1t-offline/l1t-rct-proposed-bugfix

take RCT parameters from GT

smorovic committed:
*removed eventHigh in FRD v4 header. This is 24-bit L1 event ID and thus never requires more than 32 bit entry
*added Version vs. header size lookup table to FRDEventMessage header and migrate all code to using it.

cmsbuild committed:
Merge pull request #8745 from bsunanda/Run2-alca8

bsunanda:Run2-alca8   Latest changes to meet calibration codes for HCAL

diguida commented:
@nilina please rebase

mmusich commented:
@nilina this does not merge anymore, please rebase

cmsbuild commented:
Pull request #8286 was updated. @diguida, @cerminar, @cmsbuild, @franzoni, @nclopezo, @mmusich, @davidlange6 can you please check and sign again.

Martin-Grunewald committed:
Catch user error in configuring Trigger summary

perrotta committed:
further fixes and updates

threus committed:
DQM to DQMIO

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

makortel committed:
Message logging to LogDebug/LogTrace

davidlange6 committed:
Merge pull request #8885 from ggovi/new-conddb-fix-essource-input-params

Various fixes for ConddbEssource input parameters handling

cmsbuild committed:
Merge pull request #8865 from suchandradutta/BadChannel_75X

 added LogMessage in SiStripQualityHotStripIdentifierRoot

jshlee committed:
fixed geo settings for GeometryExtended2023Muon

jshlee committed:
fixed FW3DViewGeometry.cc and FWRPZViewGeometry.cc by hardcoding gem
and me0 layout

capalmer85 committed:
remove duplication of pathALCARECOLumiPixels and link properly to pathALCARECOLumiPixelsMinBias

davidlange6 committed:
Merge pull request #8916 from cms-tsg-storm/hltIntegrationContinued75X

HLT menu updates on top of what was in CMSSW_7_4_1

cmsbuild committed:
Merge pull request #8783 from heppye/CMSSW_7_5_X

Create Bu_Mixing.dec

davidlange6 committed:
add hcalnzsRun2 to the list of run tests

Commit:
explicitly set setNegativeFlags to False, by Slava's request

cmsbuild commented:
-1
@nilina This pull request cannot be automatically  merged, could you please rebase it?

nilina commented:
Could you please tell me how I can rebase it?

Commit:
Tune and test several AlCaReco jobs and prepare DQM plots for IsoTrack Trigger

capalmer85 committed:
add bunch crossing as index to pixel counters

Commit:
Update alphaT DQM paths

cmsbuild committed:
Merge pull request #8933 from threus/to_dqmio_75x

DQM to DQMIO

davidlange6 committed:
Merge pull request #8855 from pietverwilligen/RPC4RE11_75X_PR

Separate 2015 Geometry including the 4 installed RE1/1 chambers

nclopezo commented:
Hi @nilina,
You can see instructions here: https://cms-sw.github.io/tutorial-forward-port.html

davidlange6 committed:
Merge pull request #8876 from trocino/MediumMuonID75X

New Medium muon ID selectors and offline DQM plots

Commit:
Removed offending alphaT path that was left in

Commit:
Modify path names and binnings.

silviodonato committed:
HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc

makortel committed:
Check if parametersDefinerTP is for cosmics only once and use the resulting bool later

hatakeyamak committed:
Use Configuration.StandardSequences.Reconstruction_Data_cff instead of Reconstruction_cff

makortel committed:
Tidy up dEdx treatment in MultiTrackValidator

- Remove string comparisons
- Remove copying of ValueMaps
- Tidy up monitoring implementation

Using the same ValueMap for all input track collections works for
track validation, as all input collections are Refs to the original
generalTracks.

makortel committed:
Tune TrackingParticleSelector for better performance

- Break as soon as know that the return value is false
- Reorder tests such that lighter ones are checked first
- Reduce repeated calculations (minor)

capalmer85 committed:
add event content output and fix bugged import cff line in streams

makortel committed:
Cleanup in MultiTrackValidator

Reduce repeated computations in
- MTVHistoProducerAlgoForTracker::fill_simAssociated_recoTrack_histos()
- MTVHistoProducerAlgoForTracker::fill_generic_recoTrack_histos()
- MTVHistoProducerAlgoForTracker::fill_ResoAndPull_recoTrack_histos()

Streamline MTVHistoProducerAlgoForTracker::fill_generic_recoTrack_histos(),
plus other cleanups.

silviodonato committed:
python/HltBtagPostValidation_cff.py

Commit:
Allow some debugging

silviodonato committed:
../python/HltBtagValidationFastSim_cff.py

cmsbuild committed:
Merge pull request #8947 from dgonzal/HcalResponeTune

Hcal respone paramters

davidlange6 committed:
Merge pull request #8814 from namapane/MFConfigFromDB

Set StandardSequences to take MF geometry and configuration from the DB

davidlange6 committed:
Merge pull request #8942 from davidlange6/add_hcalnzsrun2_unit

add hcalnzsRun2 to the list of run tests in Config/DataProcessing

makortel committed:
Do the TP selection for efficiency denominator only once

silviodonato committed:
HltBtagValidation_cff.py

Commit:
removed deprecated files

Commit:
Changed to LogDebug

silviodonato committed:
HltBtagValidation_cff.py

silviodonato committed:
HltBtagValidationFastSim_cff.py

rovere committed:
Moved test configuration files into a more appropriate folder, avoiding the creation of a new package.

silviodonato committed:
HLTBTagPerformanceAnalyzer.cc

rovere committed:
Add the improved reco-material description XML file under the SimTracker/TrackerMaterialAnalysis/test, temporarily, waiting for a coming PR with the properly updated XML and a properly updated payload in the database.

rovere committed:
Remove useless python configuration files.

capalmer85 committed:
BUG fix... need to reset firstEvent to false after first event

bartokm committed:
newDB Object

Martin-Grunewald committed:
FastSim cleanup

cmsbuild committed:
Merge pull request #8954 from fojensen/V0Branch

removed deprecated files

cmsbuild committed:
Merge pull request #8944 from capalmer85/pcc-ntupler-75X-v1

add bunch crossing as index to pixel counters

cmsbuild committed:
Merge pull request #8907 from smorovic/FRD-75X

Replacing adler32 with CRC32-C in FRD (raw data format) v5 - 75X

cmsbuild committed:
Merge pull request #8811 from apfeiffer1/conddb-nocol

conddb: switch to no-color mode when editing

cmsbuild committed:
Merge pull request #8956 from Martin-Grunewald/HLTFastSimCeanUp

HLT FastSim cleanup

makortel committed:
Precalculate (efficiency) TP momentum and vertex wrt PCA

This reduces the amount of repeated calculations of those. Eliminating
all repeated calculations would require fancier caching, or
precalculating them for all TPs (such that they would be accessible
through Refs, taking into account that label_tp_effic and
label_tp_fake may point to different TP collections...).

bartokm committed:
removing unused stuff

cmsbuild committed:
Merge pull request #8951 from rovere/recoMaterialEffectsCleaned

Reco material effects: updates to tools

davidlange6 committed:
Merge pull request #8350 from kodolova/physim_pedestal2015

HCAL Physim pedestal2015

cmsbuild committed:
Merge pull request #8923 from igv4321/negative-energy-filter-fix

Updated implementation of the HCAL negative energy filter

cmsbuild committed:
Merge pull request #8797 from kpedro88/JECDB-75X-1

update JEC DB writer to make path configurable and use FileInPath consistently

cmsbuild committed:
Merge pull request #8795 from ahinzmann/addCaloJetPt

Add CaloJet pT to MiniAOD

cmsbuild committed:
Merge pull request #8729 from ianna/hcal-parameters-payload-object-type

Hcal Parameters Payload Object Type

ggovi committed:
Fix for GT processing on v1 database

smorovic committed:
return false if LS limit is below the last open LS seen in fu.lock

cmsbuild committed:
Merge pull request #8962 from ggovi/new-conddb-fix-for-autocond-v1

Fix for GT processing on v1 database

cmsbuild committed:
Merge pull request #8950 from bsunanda/Run2-alca12

bsunanda:Run2-alca12 Allow some debugging to the IsoTrack trigger filter code

cmsbuild committed:
Merge pull request #8948 from capalmer85/pixel-vertex-counting-ALCAReco-v1

Pixel vertex counting alca reco v1

cmsbuild committed:
Merge pull request #8774 from ianna/update-payload-scripts-75x

Update Payload Scripts

cmsbuild committed:
Merge pull request #8834 from diguida/fix_PixelObjectsInDropBox

Added SiPixel class in the utilities for the dropbox.

mrcarver committed:
Updated DQM modules for HCAL M3

jshlee committed:
Merged CMSSW_7_5_X from repository cms-sw

civanch committed:
make MT mode default

jshlee committed:
Merged fireworks-gem-75x from repository jshlee

Conflicts:
	Fireworks/Geometry/python/dumpSimGeometry_cfg.py

pastika committed:
Fixed bug in RunRVMacro.csh

pastika committed:
Merge branch 'master' of github.com:BaylorCMS/HcalRelVal

bartokm committed:
dropbox_utilities

nilina committed:
Merged Correction-AlCaReco-HcalIterPhiSym from repository nilina

Conflicts:
	Configuration/StandardSequences/python/AlCaRecoStream_Specials_cff.py

cmsbuild committed:
Merge pull request #8964 from civanch/g4mt_default

Make MT mode default

cmsbuild committed:
Merge pull request #8949 from bendavid/ecalpfcluscorHighEnergyFix

Fix for high energy behaviour of ecal pfcluster corrections

cmsbuild committed:
Merge pull request #8669 from nickmccoll/Remove-SiStripDetId-From-CalbTracker

Remove SiStripDetId from CalibTracker/StripESProducers

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

archiron committed:
folder web page modifications

archiron committed:
folder web page modifications

archiron committed:
folder web page modifications

archiron committed:
folder web page modifications

Commit:
new Global Tags including PTracker parameters for geometry

makortel committed:
Inline all trivial accessors of TrackingParticle for better performance

nilina commented:
I followed the instructions from the tutorial. Could you please whether I do it correctly.

cmsbuild commented:
Pull request #8286 was updated. @Dr15Jones, @cvuosalo, @emeschi, @cerminar, @cmsbuild, @ggovi, @franzoni, @diguida, @slava77, @mommsen, @mmusich, @davidlange6, @nclopezo can you please check and sign again.

hdelanno committed:
Cleaned ROOT paremeters (OutputMEsInRootFile and OutputFileName)

smorovic committed:
*assert if unpacking segment is larger than remaining event size
*fedt_t instead of fedh_t (they are currently of same length)
*exception if FED ID is out of range

cmsbuild commented:
-1 
Tested at: 402e87434fcdaefd0cb3a46b40c481d7bbd4db73
 When I ran the RelVals I found an error in the following worklfows: 
 4.22 step2
<pre>runTheMatrix-results/4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/step2_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC.log

</pre>
4.53 step3
<pre>runTheMatrix-results/4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step3_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT.log

</pre>
8.0 step3
<pre>runTheMatrix-results/8.0_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS/step3_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS.log

</pre>
1000.0 step2
<pre>runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log

</pre>
1001.0 step2
<pre>runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD/step2_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD.log

</pre>
25.0 step5
<pre>runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECO+HARVEST+ALCATT/step5_TTbar+TTbar+DIGI+RECO+HARVEST+ALCATT.log

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/4549/summary.html 

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/402e87434fcdaefd0cb3a46b40c481d7bbd4db73
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/4549/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/4549/git-merge-result

Commit:
Move plugin header file to the plugin directory

cmsbuild commented:
-1 
Tested at: 402e87434fcdaefd0cb3a46b40c481d7bbd4db73
 When I ran the RelVals I found an error in the following worklfows: 
 4.22 step2
<pre>runTheMatrix-results/4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/step2_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC.log

</pre>
4.53 step3
<pre>runTheMatrix-results/4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step3_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT.log

</pre>
8.0 step3
<pre>runTheMatrix-results/8.0_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS/step3_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS.log

</pre>
1000.0 step2
<pre>runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log

</pre>
1001.0 step2
<pre>runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD/step2_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD.log

</pre>
25.0 step5
<pre>runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECO+HARVEST+ALCATT/step5_TTbar+TTbar+DIGI+RECO+HARVEST+ALCATT.log

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/4546/summary.html 

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/402e87434fcdaefd0cb3a46b40c481d7bbd4db73
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/4546/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8286/4546/git-merge-result

Commit:
Add an option to TrackingTruthAccumulator to save a collection of initial vertices

mmusich committed:
Fixing import script for DTRecoConditions

slava77 commented:
-1

AlCa step is broken 

```
StandardSequences/AlCaRecoStream_Specials_cff.py", line 48, in <module>
    from Calibration.EcalAlCaRecoProducers.ALCARECOEcalCalPhiSym_cff import *
ImportError: No module named ALCARECOEcalCalPhiSym_cff

```

cmsbuild committed:
Merge pull request #8970 from audrium/autoCond_75X_and_PTracker

new Global Tags including PTracker parameters for geometry

cmsbuild committed:
Merge pull request #8936 from jshlee/fireworks-gem-75x

Fireworks gem 75x

wmtan committed:
Secondary input source chooses first file to open randomly

davidlange6 committed:
Merge pull request #8888 from makortel/trackingVertexValidation_harmonizeTP_v5

Use common track-TP association in track and vertex validation

cmsbuild committed:
Merge pull request #8946 from ndaci/ExoValDev_75X_PR5

Update path names and binnings.

cmsbuild committed:
Merge pull request #8978 from wmtan/RandomizeFirstPileupFile

Secondary input source chooses first file to open randomly

mmusich commented:
-1 
@nilina it seem like you rebased before this commit https://github.com/bmarzocc/cmssw/commit/9a8d1652415da040d16ccf792581289d9e0aaef8 in PR #8678 from @bmarzocc removing the ECAL ALCARECOEcalCalPhiSym. Not completely clear to me how this happened, but please start again from a clean branch

Commit:
Modify in view of disappearing geometry file

Commit:
Change global tag as suggested by Salvatore

wmtan committed:
Temporary fix to clear maps each event

Commit:
Change global tag as suggested by Salvatore

davidlange6 committed:
Merge pull request #8990 from wmtan/TrackerHitAssociatorMemoryFix

Temporary fix to clear TrackerHitAssociator maps each event

wmtan committed:
Better consumes interface for TrackerHitAssociator

makortel committed:
Fix clang compilation error

hengne committed:
turn on GEN-SIM recycle for SMS-T1tttt_mGl-1500_mLSP-100_13 for next release

hengne committed:
comment out LM1_sfts_13 that replaced by SMS_T1tttt_xx

gzevi committed:
Changes to Egamma/Electrons plots: separate +/- endcap for alignment, add sieie and PFiso

ggovi committed:
Storing payloads with the related serialization info. Improved error handling in serialization.

cmsbuild committed:
Merge pull request #8989 from mark-grimes/addInitialVertices75X

Add option to save initial vertex collection (forward port of #8975)

cmsbuild committed:
Merge pull request #8976 from mmusich/fixCondCore_for_MF

Fixing import script for DTRecoConditions

cmsbuild committed:
Merge pull request #8973 from silviodonato/DQMbtagfix

HLT Btag validation code update

Commit:
Correct the cfg python scripts

wddgit committed:
Make callWhenNewProductsRegistered public

This is needed for the consumes migration of
L1GtUtils. It will make testing the other more
significant parts of that migration easier if
this is integrated quickly. This change is almost
trivial.

davidlange6 committed:
Merge pull request #9019 from wddgit/consumesMigrationL1

Make callWhenNewProductsRegistered public

ggovi committed:
Fix for conflicts caused by boost version header

hengne committed:
fix crash in workflow FS_SMS-T1tttt_mGl-1500_mLSP-100_13_PU25

hengne committed:
Merge branch 'CMSSW_7_5_X' into from-CMSSW_7_5_0_pre4_relval_forMerge

lihux25 committed:
Merge branch 'master' of https://github.com/BaylorCMS/HcalRelVal into relval_update_May09_2015

Conflicts:
	.gitignore
	DQMOffline/Hcal/python/HcalDQMOfflinePostProcessor_cff.py
	DQMOffline/Hcal/src/HcalRecHitsAnalyzer.cc
	Validation/CaloTowers/test/macros/InputRelVal_High.txt
	Validation/CaloTowers/test/macros/InputRelVal_Low.txt
	Validation/CaloTowers/test/macros/InputRelVal_Medium.txt
	Validation/CaloTowers/test/macros/InputRelVal_Pileup.txt
	Validation/CaloTowers/test/macros/RelValMacro.C
	Validation/CaloTowers/test/macros/RunRVMacros.csh
	Validation/CaloTowers/test/macros/RunRVMacros_Pileup.csh
	Validation/CaloTowers/test/macros/SinglePi.C
	Validation/CaloTowers/test/macros/html_indices/SinglePiScan.html
	Validation/CaloTowers/test/macros/rootlogon.C
	Validation/CaloTowers/test/run_onRelVal_data_cfg.py
	Validation/HcalRecHits/src/HcalRecHitsValidation.cc

lihux25 committed:
Delete README.md

lihux25 committed:
Update .gitignore

cmsbuild committed:
Merge pull request #9013 from ggovi/new-conddb-serialization-info

Storing payloads with the related serialization info. Improved error …

cmsbuild committed:
Merge pull request #9010 from gzevi/dqmoffline_egamma_updates

Changes to Egamma/Electrons plots: separate +/- endcap for alignment,…

cmsbuild committed:
Merge pull request #8999 from wmtan/BetterConsumesInterfaceForTrackerHitAssociator

Better consumes interface for TrackerHitAssociator

cmsbuild committed:
Merge pull request #8988 from makortel/mtv_optimize

Improve MultiTrackValidator performance

smorovic committed:
to determine if lumisection has ended, check local EoLS file  if locking timeout happens

cmsbuild committed:
Merge pull request #8967 from bartokm/SiPixelDynamicInefficiencyDB

SiPixelDynamicInefficiency DB object

cmsbuild committed:
Merge pull request #8971 from hdelanno/clean_sistripDQM_75X

Cleaned unused config parameters in Strip DQM for 7_5_X

cmsbuild committed:
Merge pull request #8943 from bsunanda/Run2-alca11

bsunanda:Run2-alca11 Tune+test several AlCaReco jobs and prepare DQM plots for IsoTrack Trigger

Dr15Jones committed:
Add override keyword

Dr15Jones committed:
Fixed memory leak in SimpleDAFHitCollector

The new TrackingRecHit created from the call to rightdimension() was
never deleted since SiTrackerMultiRecHitUpdator does not take ownership
of the hits passed to it. Corrected the problem by using std::unique_ptr.

smuzaffar committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X

ggovi committed:
added new tool for testing serialization

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge pull request #9048 from ggovi/new-conddb-test-serialization-tool

added new tool for testing serialization

cmsbuild committed:
Merge pull request #9033 from smorovic/fedid-check-75X

High FED ID check (75X)

cmsbuild committed:
Merge pull request #9043 from Dr15Jones/fixMemoryLeakInSimpleDAFHitCollector

Fix memory leak in SimpleDAFHitCollector

cmsbuild committed:
Merge pull request #9025 from lihux25/relval_update_May09_2015

Hcal Relval update

cmsbuild committed:
Merge pull request #8969 from archiron/Electron_Validation_OvalFile

Electron Validation OvalFile Modifications

cmsbuild committed:
Merge pull request #8966 from mrcarver/updateDQM75X

Updated SUSY DQM modules for HCAL M3 in 75X

cmsbuild committed:
Merge pull request #8945 from aelwood/dqmUpdate75

Update alphaT DQM paths

cmsbuild committed:
Merge pull request #8913 from pohsun/fix-multiple-booking

Fix multiple booking in fillSummary.

cmsbuild committed:
Merge pull request #9026 from hengne/from-CMSSW_7_5_0_pre4_relval_forMerge

Fix crash of workflow FS_SMS-T1tttt_mGl-1500_mLSP-100_13_PU25

nilina committed:
Merged refs/pull/8286/head from repository cms-sw

nilina commented:
I tried to fix problems. Could you please check it. Thanks.

cmsbuild commented:
Pull request #8286 was updated. @diguida, @ianna, @cerminar, @covarell, @bendavid, @civanch, @cmsbuild, @davidlange6, @vciulli, @Dr15Jones, @cvuosalo, @mdhildreth, @deguio, @slava77, @ggovi, @vadler, @mmusich, @nclopezo, @danduggan, @thuer, @monttj, @franzoni can you please check and sign again.

mmusich commented:
-1
@nilina this has still issues in list of commits.
May I suggest to close it and start from scratch starting from the lastest IB?
Thanks

mmusich commented:
superseded by #9075 
@diguida thanks
